cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -1075,12 +1113,17 @@ instance (Condense (HeaderHash blk), LedgerSupportsProtocol blk)
     ChainDB.TrySwitchToAFork pt _ ->
       mkObject [ "kind" .= String "TraceAddBlockEvent.TrySwitchToAFork"
                , "block" .= toObject verb pt ]
-    ChainDB.AddedToCurrentChain _ _ c ->
+    ChainDB.AddedToCurrentChain _ base extended  ->
       mkObject [ "kind" .= String "TraceAddBlockEvent.AddedToCurrentChain"
-               , "newtip" .= showPoint verb (AF.headPoint c) ]
-    ChainDB.SwitchedToAFork _ _ c ->
-      mkObject [ "kind" .= String "TraceAddBlockEvent.SwitchedToAFork"
-               , "newtip" .= showPoint verb (AF.headPoint c) ]
+               , "newtip" .= showPoint verb (AF.headPoint extended)
+               , "headers" .= toJSON (toObject verb `map` addedHdrsNewChain base extended) ]
```

Comment:
This is expensive, since it computes intersections. We should keep it to `MaximalVerbosity` too, like you do below with `SwitchedToAFork`.

Reply:
Done!